functions/src/src/UpdatePointLogStatus.ts

```diff
@@ -79,7 +80,7 @@ export async function updatePointLogStatus(approve: boolean, approver_id: string
                     log.approveLog(user)
                     await doc_ref.set(log.toFirebaseJSON())
                     await addPoints(parseInt(point_value), user.house, resident_id)
-                    //await addPoints(5, user.house, resident_id)
+                    await addPoints(5, user.house, resident_id)
```

Comment:
Does this always add 5 points to the user's house?

Reply:
Whoops looks like the line got uncommented instead of deleted. Good catch. I had it always doing 5 points during an early stage of testing.